lib/zlib.js

```diff
@@ -99,7 +99,7 @@ function zlibBuffer(engine, buffer, callback) {
     var chunk;
     while (null !== (chunk = engine.read())) {
       buffers.push(chunk);
-      nread += chunk.length;
+      nread += chunk.byteLength;
```

Comment:
I think so, but I'd rather err on the side of replacing too many `Buffer.length`s rather than missing an `ArrayBufferView.byteLength`, since `Buffer.length === Buffer.byteLength`.